main/ui/src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java

```diff
@@ -52,13 +53,19 @@
 	public RadioButton mountPointCustomDir;
 	public ChoiceBox<String> driveLetterSelection;
 
+	//FUSE + Windows -> Disable some (experimental) features for the user because they are unstable
+	//Use argument Dfuse.experimental="true" to override
+	private final BooleanBinding fuseAndWindows;
```

Comment:
Please rename the variable, it is misleading.
Suggestion: `restrictToStableFuseOnWindows`

Reply:
Resolved by 528510eb53be6ffd7989fe67f0b6f5cc56ffa904.